nbody/sample_workunits/for_developers.lua

```diff
@@ -209,8 +295,11 @@ end
 function get_soft_par()
     --softening parameter only calculated based on dwarf,
     --so if manual bodies is turned on the calculated s.p. may be too large
-    sp = calculateEps2(totalBodies, rscale_l, rscale_d, mass_l, mass_d, UseOldSofteningLength)
-
+    if (UseOldSofteningLength == 1) then
+        sp = calculateEps2(totalBodies, rscale_l, rscale_d, mass_l, mass_d)
+    else
+        sp = calculateEps2_NEW(comp1, totalLightBodies)
```

Comment:
Instead of `NEW` perhaps use something that describes what it does differently.  Calling it `Ethan` probably isn't quite right, but something along that line of thinking would be better.